wrapper.js

```diff
@@ -50,6 +50,23 @@ function setupMethods (soljson) {
     } else {
       result = compileJSON(input, optimise);
     }
+
+    var res = JSON.parse(result);
```

Comment:
Yep, I agree. I will change that and move it out of the wrapper, no problem.
Regarding the splitting, the changes are rather related, it will be hard to split the changes.